react/useVideo.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
I suggest moving this to the line in which you're actually breaking this rule. And also add a comment explaining why it is ok to do so.